node/toolrunner.ts

```diff
@@ -1022,6 +1023,16 @@ export class ToolRunner extends events.EventEmitter {
         res.stderr = (r.stderr) ? r.stderr.toString() : '';
         return res;
     }
+
+    /**
+     * Used to close child process by sending SIGNINT signal.
+     * It allows to executed script to have some aditional logic on SIGINT, before exiting.
+     */
+    public killChildProcess() {
```

Comment:
Nit: could you add ':void' as return type?

Reply:
Yeah, thanks, added